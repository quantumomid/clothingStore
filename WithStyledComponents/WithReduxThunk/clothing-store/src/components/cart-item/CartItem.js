// import "./CartItem.scss";

import { CartItemContainer, CartItemImage, ItemDetailsContainer } from "./cartItemStyles";

const CartItem = ({ item: { imageUrl, price, name, quantity } }) => {
    return (
        <CartItemContainer>
            <CartItemImage src={imageUrl} alt="item" />
            <ItemDetailsContainer>
                <span>{name}</span>
                <span>{quantity} x Â£{price}</span>
            </ItemDetailsContainer>
        </CartItemContainer>
    )
}

export default CartItem;