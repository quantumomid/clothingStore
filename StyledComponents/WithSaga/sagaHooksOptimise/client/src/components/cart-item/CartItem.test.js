import { shallow } from "enzyme";
import CartItem from "./CartItem";


it("Cart Item component should be rendered", () => {
    const mockItem =    {
        id: 3,
        name: 'Brown Cowboy',
        imageUrl: 'https://i.ibb.co/QdJwgmp/brown-cowboy.png',
      //   imageUrl: "/images/shop-img/hats/brown-cowboy.png",
        price: 35
      }

      expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
})