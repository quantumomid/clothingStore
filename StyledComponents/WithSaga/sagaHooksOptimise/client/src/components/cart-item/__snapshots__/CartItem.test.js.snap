// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart Item component should be rendered 1`] = `ShallowWrapper {}`;
